lizard_auth_server/forms.py

```diff
@@ -4,7 +4,7 @@
 
 from django import forms
 from django.conf import settings
-from django.contrib import auth
+from django.contrib.auth import forms as authforms
```

Comment:
Die `as authforms`, is dat iets dat standaard aangeraden wordt? Danwel "waarom deze verandering, is het nodig?" :-)